src/charts/donut/donut-pct-chart-directive.js

```diff
@@ -0,0 +1,354 @@
+       center-label =<br> 'available'
+     </div>
+     <div class="col-md-3">
+       <div pf-donut-pct-chart id="pct" config="pctConfig" data="pctData" center-label="percent"></div>
+       center-label =<br> 'percent'
+     </div>
+     <div class="col-md-3">
+       <div pf-donut-pct-chart id="none" config="noneConfig" data="noneData" center-label="none"></div>
+       center-label =<br> ' none'
+     </div>
+
+     <div class="col-md-12">
+       <hr>
+     </div>
+
+     <div class="col-md-12">
+       Custom Tooltip and Center Label
+       <div pf-donut-pct-chart id="cust" config="custConfig" data="custData"></div>
+     </div>
+   </div>
+</file>
+
+<file name="script.js">
+   function ChartCtrl($scope) {
+
+     $scope.config = {
+       'units': 'GB',
+       'thresholds':{'warning':'60','error':'90'}
+     };
+
+     $scope.data = {
+         'used': '950',
+         'total': '1000'
+     };
+
+     $scope.newUsed = $scope.data.used;
+
+     $scope.$watch('newUsed', function(val) {
+       $scope.data.used = val;
+     });
+
+     $scope.usedConfig = {
+       'units': 'GB',
+       'thresholds':{'warning':'60','error':'90'}
+     };
+
+     $scope.usedData = {
+         'used': '350',
+         'total': '1000'
+     };
+     $scope.availConfig = {
+       'units': 'GB',
+       'thresholds':{'warning':'60','error':'90'}
+     };
+
+     $scope.availData = {
+         'used': '350',
+         'total': '1000'
+     };
+     $scope.pctConfig = {
+       'units': 'GB',
+       'thresholds':{'warning':'60','error':'90'}
+     };
+
+     $scope.pctData = {
+         'used': '350',
+         'total': '1000'
+     };
+     $scope.noneConfig = {
+       'units': 'GB',
+       'thresholds':{'warning':'60','error':'90'}
+     };
+
+     $scope.noneData = {
+         'used': '350',
+         'total': '1000'
+     };
+
+     $scope.custConfig = {
+       'units': 'MHz',
+       'thresholds':{'warning':'60','error':'90'},
+       "legend":{"show":true},
+       'tooltipFn': function (d) {
+         return '<span class="donut-tooltip-pf"style="white-space: nowrap;">' +
+                   d[0].value + ' ' + d[0].name +
+                '</span>';
+       },
+       'centerLabelFn': function (scope) {
+         return '<tspan dy="0" x="0" class="donut-title-big-pf">' +
+                  scope.data.available + '</tspan>' +
+                '<tspan dy="20" x="0" class="donut-title-small-pf">' +
+                  'Free' +
+                '</tspan>';
+       }
+     };
+
+     $scope.custData = {
+         'used': '670',
+         'total': '1000'
+     };
+
+   };
+ </file>
+ </example>
+ */
+angular.module('patternfly.charts').directive('pfDonutPctChart', ['ChartsMixin', '$timeout',
+  function(chartsMixin, $timeout) {
+    'use strict';
+    return {
+      restrict: 'A',
+      scope: {
+        id: '@',
+        config: '=',
+        data: '=',
```

Comment:
I think it's best practice to use prefixed attributes (i.e. `chart-id`, `chart-data`) so as to avoid a potential collision with actual HTML elements.  If you want to use the html ID of the element you could use the `$attrs` parameter that is passed into the controller.